spec.md

```diff
@@ -649,6 +649,16 @@ $app->get('/hello/{name}', function ($name) use ($app) {
 });
 ```
 
+Method chaining SHOULD be put on separate lines, where each subsequent line is indented once. When doing so, the first
+method MUST be on the next line.
+
+```php
+$someInstance
+    ->create()
+    ->prepare()
+    ->run();
```

Comment:
Placing `;` on a separated line is handy as well.

Reply:
It's just a little handy to comment/move/remove lines when you have `;` not at the end of the chain, but I think it's unnecessary to have it as a recommendation :)

Reply:
in my opinion it is best to admit both versions :)